lib/rails_cursor_pagination/paginator.rb

```diff
@@ -64,6 +64,10 @@ def initialize(relation, first: nil, after: nil, last: nil, before: nil,
         last ||
         RailsCursorPagination::Configuration.instance.default_page_size
 
+      @page_size = [
+        @page_size,
+        RailsCursorPagination::Configuration.instance.max_page_size
+      ].compact.min
```

Comment:
While elegant, I personally find this a bit hard to read, especially also due to the `#compact` call. It also adds computational complexity every single time, although it might not be required (creating an array, compacting it, finding the minimum and then assigning it).

Therefore, what do you think about making this explicit?
```suggestion
      if Configuration.instance.max_page_size &&
         Configuration.instance.max_page_size < @page_size
        @page_size = Configuration.instance.max_page_size
      end
```

It's a lot of calls to `Configuration.instance.max_page_size`, but they shouldn't "cost" much 🙃